docs/dev/tests.rst

```diff
@@ -40,20 +40,23 @@ you can also set the ``TOX_POSARGS`` environment variable to an empty string:
 
 .. prompt:: bash
 
-       tox -- -m 'not search' -x
+   tox -- -m 'not search' -x
 
 To target a specific environment:
 
 .. prompt:: bash
 
-    tox -e py38
+   tox -e py38
 
 The ``tox`` configuration has the following environments configured. You can
 target a single environment to limit the test suite:
 
 py38
     Run our test suite using Python 3.8
 
+py38-debug
+    Same as ``py38``, but there are some useful debugging tools available in the environment.
```

Comment:
Is it worth defining what is installed? Or maybe linking to the file? Not necessary though.